components/cam/src/physics/crm/sam/fft.F90

```diff
@@ -3,8 +3,12 @@ module fft_mod
 contains
 
       subroutine fft991_crm(a,work,trigs,ifax,inc,jump,n,lot,isign)
-      !$acc routine seq
       use params, only: crm_rknd
+#if defined(_OPENACC)
```

Comment:
I believe the placement of `!$acc routine seq` matters for the PGI compiler. Please move `!$acc routine seq` back to the original location for OpenACC for all routines that use it.